applications/CoSimulationApplication/python_scripts/solver_wrappers/rigid_body/rigid_body_input_check.py

```diff
@@ -0,0 +1,176 @@
+
+import KratosMultiphysics as KM
+
+import json
+from importlib import import_module
+
+
+"""
+This file checks the input for the RigidBodySolver and raises exceptions if necessary.
+"""
+
+
+def _CheckMandatoryInputParameters(project_parameters):
+    # Function to raise exceptions if some mandatory parameters were not given
+
+    # The project parameters need to have at least these base fields
+    for key in ["problem_data", "solver_settings", "output_processes", "processes"]:
+        if not project_parameters.Has(key):
+            msg = 'The key "' + key + '" was not found in the project parameters '
```

Comment:
```suggestion
            msg = f'The key "{key}" was not found in the project parameters '
```

I suggest to use f-strings